util/reggen/reg_top.sv.tpl

```diff
@@ -418,7 +418,8 @@ ${field_sig_decl(f, sig_name, r.hwext, r.shadowed, r.async_clk)}\
     .dst_wd_o     (${dst_wd_expr})
   );
       % if r.needs_we():
-  assign unused_${base_name}_${r_name}_wdata = ^${base_name}_${r_name}_wdata;
+  assign unused_${base_name}_${r_name}_wdata =
+    ^${base_name}_${r_name}_wdata;
```

Comment:
This is a super nit (sorry) but I believe we do 4 spaces on [wrapped lines](https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md#line-wrapping).

Reply:
ah thanks for the reminder.